### PR TITLE
Create React App Cypress Code Coverage Scripts

### PR DESCRIPTION
**Pull request by kevinold**
For UNIFY-74

Associated with 
#63 
#64 
#65 
#66 
#67

### CODE REVIEW

File: scripts/create-react-app/pr.md [hunk 1]
```diff
@@ -11,6 +11,12 @@
    6. `npx cypress open-ct` (Notice that the fonts don't inherit global app styles)
    7. Edit [cypress/support/index.js](cypress/support/index.js) to import global app styles, the Cypress test preview should update automatically
    8. ${COMMIT_1}
+3. Add Cypress Code Coverage
+   1. ${COMMAND_3}
+   2. Edit [cypress.json](cypress.json) to enable `coverage`
+   3. Edit [cypress/plugins/index.js](cypress/plugins/index.js) to configure the Cypress code coverage task
+   4. Edit [cypress/support/commands.js](cypress/support/commands.js) to import Cypress code coverage support
```
Comment (cowboy) [inline scripts/create-react-app/pr.md:18]:
```suggestion
   4. Edit [cypress/support/index.js](cypress/support/index.js) to import Cypress code coverage support
```

File: scripts/create-react-app-react-router/pr.md [hunk 2]
```diff
@@ -15,3 +15,9 @@
    6. `npx cypress open-ct` (Notice that the fonts don't inherit global app styles)
    7. Edit [cypress/support/index.js](cypress/support/index.js) to import global app styles, the Cypress test preview should update automatically
    8. ${COMMIT_2}
+4. Add Cypress Code Coverage
+   1. ${COMMAND_4}
+   2. Edit [cypress.json](cypress.json) to enable `coverage`
+   3. Edit [cypress/plugins/index.js](cypress/plugins/index.js) to configure the Cypress code coverage task
+   4. Edit [cypress/support/commands.js](cypress/support/commands.js) to import Cypress code coverage support
```
Comment (cowboy) [inline scripts/create-react-app-react-router/pr.md:22]:
```suggestion
   4. Edit [cypress/support/index.js](cypress/support/index.js) to import Cypress code coverage support
```

File: scripts/create-react-app-redux/pr.md [hunk 3]
```diff
@@ -14,3 +14,9 @@
    6. `npx cypress open-ct` (Notice that the fonts don't inherit global app styles)
    7. Edit [cypress/support/index.js](cypress/support/index.js) to import global app styles, the Cypress test preview should update automatically
    8. ${COMMIT_2}
+4. Add Cypress Code Coverage
+   1. ${COMMAND_3}
+   2. Edit [cypress.json](cypress.json) to enable `coverage`
+   3. Edit [cypress/plugins/index.js](cypress/plugins/index.js) to configure the Cypress code coverage task
+   4. Edit [cypress/support/commands.js](cypress/support/commands.js) to import Cypress code coverage support
```
Comment (cowboy) [inline scripts/create-react-app-redux/pr.md:21]:
```suggestion
   4. Edit [cypress/support/index.js](cypress/support/index.js) to import Cypress code coverage support
```

File: scripts/create-react-app-tailwind/pr.md [hunk 4]
```diff
@@ -17,3 +17,9 @@
    6. `npx cypress open-ct` (Notice that the fonts don't inherit global app styles)
    7. Edit [cypress/support/index.js](cypress/support/index.js) to import Tailwind and global app styles, the Cypress test preview should update automatically
    8. ${COMMIT_3}
+4. Add Cypress Code Coverage
+   1. ${COMMAND_3}
+   2. Edit [cypress.json](cypress.json) to enable `coverage`
+   3. Edit [cypress/plugins/index.js](cypress/plugins/index.js) to configure the Cypress code coverage task
+   4. Edit [cypress/support/commands.js](cypress/support/commands.js) to import Cypress code coverage support
```
Comment (cowboy) [inline scripts/create-react-app-tailwind/pr.md:24]:
```suggestion
   4. Edit [cypress/support/index.js](cypress/support/index.js) to import Cypress code coverage support
```

**Review by cowboy — CHANGES_REQUESTED** [review]
Overall, looks great, but I caught a few typos that will require having to re-generate PRs #63 #64 #65 #66 #67

One general note: should we add a bullet point to each "Add Cypress Code Coverage" section about modifying the `.gitignore` file? If we think that's important, I'd like to include it.

Also, can you use the generated "PR BODY" from the script somewhere in your PR, so it's easier to click into the generated README for review? Doing that might help us catch typos like the ones I noticed in this review!

**Comment by kevinold** [comment]
@cowboy Per slack conversation, the CRA scripts are updated https://github.com/cypress-io/cypress-component-testing-examples/pull/62/commits/acfe99271aade2ef334c49652e8cfb309101349a and PRs #63 #64 #65 #66 #67 re-generated

**Review by cowboy — APPROVED** [review]

**Pull request merged**